### DISCUSSION TITLE
Prisma Accelerate Not Pushing to Database

### DISCUSSION THREAD

**Discussion by CardCounter21** [prisma/prisma#23664] [Q&A] [answered]
### Question

Hello all! 

I have a Next.js project using prisma accelerate that connects to my planetscale database. I have been working on my Schema but whenever I try to push the changes to the databse I get this error: 

Han@Hans-MacBook Card-Counter % npx prisma db push
Environment variables loaded from .env
Prisma schema loaded from prisma/schema.prisma
Datasource "db": MySQL database "card-counter" at "aws.connect.psdb.cloud"
Error: direct DDL is disabled
   0: sql_schema_connector::apply_migration::migration_step
           with step=CreateTable { table_id: TableId(1) }
             at schema-engine/connectors/sql-schema-connector/src/apply_migration.rs:21
   1: sql_schema_connector::apply_migration::apply_migration
             at schema-engine/connectors/sql-schema-connector/src/apply_migration.rs:10
   2: schema_core::state::SchemaPush
             at schema-engine/core/src/state.rs:433

I have accelerate enabled however I also have Direct Database url that connects directly to my planetscale database in my .env file. My prisma.schema is also set up to be able to take in direct pushing to my database. 

I do not know why it is not working, when I try and migrate it does not work either. I will attatch screenshots of everything. Thank you so much and have a great day everyone!

schema: <img width="1470" alt="Screenshot 2024-03-29 at 8 21 10 PM" src="https://github.com/prisma/prisma/assets/160655497/5f1030bb-55a1-453d-9e31-60ee69878b32">

Error Message: 
<img width="691" alt="Screenshot 2024-03-29 at 8 21 47 PM" src="https://github.com/prisma/prisma/assets/160655497/1b8ba761-7793-465c-8a5b-46e0ab968841">

### How to reproduce (optional)

npx prisma db push

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by janpio** [comment, marked as answer]
Your Planetscale database is not configured to allow changing database schema changes.
See: https://planetscale.com/docs/concepts/branching#how-to-make-schema-changes-on-a-branch-with-safe-migrations-enabled

**Reply by CardCounter21** [reply]
Got it ! Created a dev branch with safe migrations disabled! Thank you so much for your help!
